main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -83,17 +84,14 @@
 import org.apache.commons.collections4.MapUtils;
 import org.apache.commons.lang3.ArrayUtils;
 import org.apache.commons.lang3.StringUtils;
+import org.json.JSONException;
 
 public class DataStore {
 
     public static final String DB_FILE_NAME = "data";
     public static final String DB_FILE_NAME_BACKUP = "cgeo.sqlite";
     public static final String DB_FILE_CORRUPTED_EXTENSION = ".corrupted";
 
-    private DataStore() {
```

Comment:
While it is an improvement, it is not related to this pull request. But IMHO this change can stay because you were working in this area.